app/javascript/components/layout/top-navbar-component.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <nav class="bg-gray-800">
+    <div class="max-w-7xl px-2 sm:px-6 lg:px-8">
+      <div class="relative flex items-center justify-between h-16">
+        <div class="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start">
+          <div class="flex-shrink-0 flex items-center">
+            <a
+              href="/"
+              class="text-white px-3 py-2 rounded-md text-lg font-bold"
+            >SuperKitchen</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </nav>
+</template
+
+<script
+>
+
+export default {
+}
+</script>
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline required at end of file but not found          eol-last